test/integration/basic_test.go

```diff
@@ -106,14 +106,22 @@ func compareExpectedWithActualContains(expected string, actual string) error {
 }
 
 func compareExpectedWithActualEquals(expected string, actual string) error {
-
 	if actual != expected {
 		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
 	}
 
 	return nil
 }
 
+func findExpectedMatchInActual(expectedMatch, actual string) error {
+	r, _ := regexp.Compile(expectedMatch)
+	if !r.MatchString(actual) {
+		return fmt.Errorf("Actual output %s doesn't contain match %s", actual, expectedMatch)
```

Comment:
'ExpectedMatchInActual' => 'match in' means the same as 'contain'

and then 'match' and 'contain match' are not the same thing... ??? is it not just contain???